### PR TITLE
Landing page

### PR DESCRIPTION
**Pull request by ritikaawasthy**

### CODE REVIEW

**Review by sanjitsarkar — APPROVED** [review]
Looks good but make sure to remove whitespaces

File: documentation/styles.css [hunk 1]
```diff
@@ -1,73 +1,79 @@
...
-  "side main";
-  /* "si foot foot foot"; */
-   height:100vh;
-   grid-template-columns: 20rem 1fr ;
-   grid-template-rows: 4rem auto auto auto ;
+  grid-template-areas: "head"
+  "main"
+  "foot";
+  grid-template-columns: 1fr;
+  grid-template-rows: 4rem 1fr 100px;
```
Comment (zoef-yt) [inline documentation/styles.css:12]: Use rem instead of px to stay consistent.

File: index.html [hunk 2]
```diff
@@ -1,544 +1,57 @@
...
-          <h4 class="white-trans-bg">Heading</h4>
-          <span class='card-icon right'>&#10540;</span>
-        </div>
-        <div class='card-content-foot inline' >
-          <span class='badge badge-text prim'>Bagde</span>
-          <span class='badge badge-text sec'>Badge</span>
-        </div>
       </div>
-    </div>
+      <img src="css/media/hero-tree-big.gif" alt="">
```
Comment (zoef-yt) [inline index.html:39]: alt is empty

File: templates/docs.html [hunk 3]
```diff
@@ -0,0 +1,544 @@
...
+        </div>
+
+
+        <h3>1. Squared Avatar</h3>
+
+        <div class="container">
+          <img class="avatar xs" src="/css/media/avatar-cat.png">
+          <img class="avatar s" src="/css/media/avatar-cat.png">
+          <img class="avatar m" src="/css/media/avatar-cat.png">
+          <img class="avatar l" src="/css/media/avatar-cat.png">
```
Comment (zoef-yt) [inline templates/docs.html:130]: naming avatar-xs instead of just xs would be better for the future. Its just a suggestion do let me know your thought.

**Review by zoef-yt — APPROVED** [review]
Minor changes. The code looks good to me. Please do add a description to your PR's is empty. Adding images about the change would be good too. Good to merge with changes.

**Review by saurabhchirde — APPROVED** [review]
Your code is clean and easily readable.
Looks good to merge.

File: templates/docs.html (hunk 3 above)
Comment (ritikaawasthy) [reply templates/docs.html:130]: This sizing is applied only when class  avatar is also applied,  i thought avatar avatar-s would be too wordy. let me know your thoughts
